sympy/functions/combinatorial/numbers.py

```diff
@@ -2175,6 +2175,144 @@ def eval(cls, n):
         return Integer(cls._motzkin(n - 1))
 
 
+#-----------------------------------------------------------------------------#
+#                                                                             #
+#                         Schoeder numbers                                    #
+#                                                                             #
+#-----------------------------------------------------------------------------#
+
+
+class schroeder(Function):
+    r"""
+    In mathematics, the Schroeder number, also called a
+    large Schroeder number or big Schroeder number, describes
+    the number of lattice paths from the southwest corner
+    $(0, 0)$ of an $n\times n$ grid to the northeast corner $(n, n)$ using only
+    single steps north, $(0, 1)$ northeast, $(1, 1)$ or east, $(1, 0)$
+    that do not rise above the SW-NE diagonal. The nth Schroeder number
+    is the number of Schroeder paths of length n. The Schroeder numbers were
+    named after the German mathematician Ernst Schroeder.
+
+    Schroeder numbers are the integer sequences defined by the
+    initial terms `S_0 = 1`, `S_1 = 2` and which satisfies the two-term recurrence relation
+    `S_n = \frac{6*n - 3}{n + 1} * S_{n-1} - \frac{n - 2}{n + 1} * S_{n-2}`.
```

Comment:
Use consistent line wrapping for text in the docstring

Reply:
I have made changes to the docstring but I'm not sure whether it is right way of formatting.